### PR TITLE
Release v1.0.0 to main merge

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
Merging the changes for Release v1.0.0 into the main branch.

### CODE REVIEW

**Review by ann-aot — APPROVED** [review]
👍

**Review by dheepak-aot — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ High, new features released weekly
 - [Technologies Overview](./docs/technologies-overview.md)
 - [BCeID-Web Service Integration](./docs/bceid-webservice-integration.md)
 - How To
-  - [DevOps](./docs/howto/devops.md)
+  - [DevOps](https://github.com/bcgov/SIMS/wiki/DevOps-and-Running-the-Application#devops)
   - [Database Migration](./docs/howto/db-migration.md)
-  - [Keycloak Configuration](./docs/hotwot/keycloak-config.md)
+  - [Keycloak Configuration](https://github.com/bcgov/SIMS/wiki/Keycloak-Config)
```
Comment (guru-aot) [inline README.md:28]: 👍  Now the links will work 😄

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrewsignori-aot

**Review by sh16011993 — APPROVED** [review]
👍

**Pull request merged**